tests/llm/utils/classifiers.py

```
@@ -62,107 +35,108 @@
+            input=prompt_prefix,
+            output=eval_result.metadata.get("rationale", ""),
+            expected=context,
+            scores={
+                "context": eval_result.score,
+            },
+            metadata=eval_result.metadata,
+        )
+        return eval_result
+
+
+def evaluate_correctness(
```

Ruff (0.8.2)

Missing return type annotation for public function `evaluate_correctness`

(ANN201)